builtin/providers/azurerm/import_arm_virtual_machine_scale_set_test.go

```diff
@@ -31,6 +31,29 @@ func TestAccAzureRMVirtualMachineScaleSet_importBasic(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMVirtualMachineScaleSet_importBasic_managedDisk(t *testing.T) {
+	resourceName := "azurerm_virtual_machine_scale_set.test"
+
+	ri := acctest.RandInt()
+	config := fmt.Sprintf(testAccAzureRMVirtualMachineScaleSet_basic, ri, ri, ri, ri, ri, ri, ri, ri)
```

Comment:
You might be looking at `testAccAzureRMVirtualMachineScaleSet_basicLinux_managedDisk` which has `config := fmt.Sprintf(testAccAzureRMVirtualMachineScaleSet_basicLinux_managedDisk, ri, ri, ri, ri, ri, ri)`